test/merchant_test.rb

```diff
@@ -1,34 +1,27 @@
 require 'simplecov'
 SimpleCov.start
-require './lib/merchant'
-require 'rspec'
-
-RSpec.describe Merchant do
+require './lib/helper'
+
+RSpec.describe InvoiceRepository do
+  let!(:sales_engine) {SalesEngine.from_csv({
+    :items     => "./data/items.csv",
+    :merchants => "./data/merchants.csv"
+  })}
+  let!(:merchant_repository) {sales_engine.merchants}
+  let!(:merchant) {merchant_repository.create(Merchant.new({
+    :id => 5,
+    :name => "Turing School"
+  }))}
 
   it 'exists' do
-    merchant = Merchant.new({
-      :id => 5,
-      :name => "Turing School"
-      })
-
-    expect(merchant).to be_instance_of(Merchant)
+    expect(merchant_repository).to be_instance_of(Merchant)
```

Comment:
This is currently returning the MerchantRepository so is failing, but could be changed to `expect(merchant_repository.first).to be_instance_of(Merchant)` and it should pass I think

Reply:
fixed